scripts/npm-link.sh

```diff
@@ -3,3 +3,5 @@ set -e
 
 echo "Running npm link..."
 npm link
+
+chmod u+x ./lib/bin/firebase.js
```

Comment:
extreme nitpicking, but wouldn't +rx be more appropriate here?

Reply:
Was always created with read-access, but added it to be explicit.